static/app/views/app/index.tsx

```diff
@@ -270,25 +272,40 @@ function App({children, params}: Props) {
       <LastKnownRouteContextProvider>
         <RouteAnalyticsContextProvider>
           {renderOrganizationContextProvider(
-            <AsyncSDKIntegrationContextProvider>
-              <GlobalFeedbackForm>
-                <GlobalDrawer>
-                  <MainContainer tabIndex={-1} ref={mainContainerRef}>
-                    <GlobalModal onClose={handleModalClose} />
-                    <SystemAlerts className="messages-container" />
-                    <Indicators className="indicators-container" />
-                    <ErrorBoundary>{renderBody()}</ErrorBoundary>
-                  </MainContainer>
-                </GlobalDrawer>
-              </GlobalFeedbackForm>
-            </AsyncSDKIntegrationContextProvider>
+            <ChonkThemeProvider>
```

Comment:
note: this is the top-most place where we can have access to the `OrganizationProvider`. There’s a `ThemeAndStyleProvider` at the very root of the application, but it can’t read the organization from `useOrganization`.

Another idea would’ve been to move `ThemeAndStyleProvider` down here to be under the `OrganizationProvider`, but given that the `chonk-ui` feature flag is temporary, I think adding another theme provider is the better way.